app/mattermost.android.js

```diff
@@ -0,0 +1,436 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
lol this should be 2015.

Also why this entry file does not work for iOS as well?

Reply:
👍 

iOS is currently using the old Mattermost class just for safety precautions.  Our team is easily able to test on Android.  Next is testing on iOS, but will require some help in order to get that going since it's not easy testing push notifications on iOS unfortunately